src/test/java/org/openstreetmap/atlas/utilities/filters/AtlasEntityPolygonsFilterTest.java

```diff
@@ -430,7 +431,13 @@ public void testMultiPolygonForms()
                 3, 1, -1, -1);
     }
 
+    /**
+     * Ignored because of changed functionality. Functionality a legacy of first version of filter.
+     * Originally overlapping polygons were not allowed in the case of both include polygons. Now,
+     * only one type of polygon exists in the filter at once.
+     */
     @Test
+    @Ignore
```

Comment:
Instead of ignoring, let's update it to reflect updated behavior or remove it altogether.